components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -156,6 +156,40 @@ Installations require elevated privileges, so run the commands as the superuser
 
 1. [Set up `knife`]({{< relref "infra-server.md#use-knife-with-chef-infra-server" >}}) for use with Chef Infra Server.
 
+## How to Add Chef Infra Server to a Chef Automate Installation
+
+Patch an existing Chef Automate installation to add Chef Infra Server:
+
+1. Create a `patch.toml` file to add `infra-server` to the list of products to deploy:
+
+    ```toml
+       [deployment.v1.svc]
+       products=["automate", "infra-server"]
+    ```
+
+2. Apply the patch to the Chef Automate installation:
+
+    ```shell
+       sudo chef-automate config ./patch.toml
+    ```
+   The command output shows the Chef Infra Server services being added:
```

Comment:
Markdown lint reports that there should be a blank line on either side of a fenced code block or a list.
Same (pre-existing) issue on lines 231, 233, 237, 239, and 256 in this file, and several in the other file.